package.json

```diff
@@ -54,7 +54,8 @@
     "watch": "tsc -watch -p ./",
     "pretest": "npm run compile && npm run lint",
     "lint": "eslint src --ext ts",
-    "test": "node ./out/test/runTest.js"
+    "test": "cp -R test/tutorial/. test/runTests && node ./out/test/runTest.js",
+    "testWindows": "xcopy /s .\\test\\tutorial .\\test\\runTests\\ && node ./out/test/runTest.js"
```

Comment:
Could you check whether there is a node util or something to copy files, those windows / linux path are usually quite a pain to maintain.

Reply:
I did find a node module to resolve this, thanks!